Classes/Command/EventStoreCommandController.php

```diff
@@ -113,4 +120,17 @@ public function dropSchemaCommand()
             $this->quit(1);
         }
     }
+
+    /**
+     * Forward new events to listeners
+     *
+     * This command allows you to play all relevant unseen events for all asynchronous event listeners.
+     *
+     * @return void
+     */
+    public function catchUpCommand()
```

Comment:
Yeah, I fully agree - there should be just one command for catch up and watch which applies to all relevant event listeners.

I already started streamlining that - maybe put that into a new pull request?